eqllib/analytics/credential-access/T1003-taskmgr-dump-lsass.toml

```diff
@@ -0,0 +1,18 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "high"
+contributors = ["Tony Lambert"]
+created_date = "01/07/2019"
+description = "Identifies usage of ``taskmgr.exe`` to export the memory space of lsass.exe which contains sensitive credentials."
+id = "210b4ea4-12fc-11e9-8d76-4d6bb837cda4"
+name = "LSASS Memory Dumping via TaskMgr.exe"
```

Comment:
I think this is just about ready. Last minor request -- do you mind updating the name and description to reflect the changes?